### PR TITLE
Update slack step

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -29,11 +29,22 @@ if [ -n "$DEPLOY" ]; then
 else
   # its a build!
   export ACTION="build"
-  export ACTION_URL=$WERCKER_BUILD_URL
+  export ACTION_URL=$WERCKER_RUN_URL="https://app.wercker.com/APIPCS/$WERCKER_APPLICATION_NAME/runs/build/$WERCKER_BUILD_ID"
```
Comment (nadade) [inline run.sh]: I don't understand the intention here; is this supposed to set a default value?

File: run.sh [hunk 2]
```diff
@@ -29,11 +29,22 @@ if [ -n "$DEPLOY" ]; then
...
+if [ "$WERCKER_RESULT" = "passed" ]; then
+  export RESULT="Sucess"
+fi
+
+export GIT_COMMIT_URL="https://$WERCKER_GIT_DOMAIN/$WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY/commit/$WERCKER_GIT_COMMIT"
+export MESSAGE="$RESULT: #<$ACTION_URL|$WERCKER_SHORT_BUILD_ID> for $WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY by $WERCKER_STARTED_BY on branch <$GIT_COMMIT_URL|$WERCKER_GIT_BRANCH>"
+export FALLBACK="$ACTION for $WERCKER_GIT_OWNER/$WERCKER_GIT_REPOSITORY by $WERCKER_STARTED_BY has $WERCKER_RESULT on branch $WERCKER_GIT_BRANCH"
 export COLOR="good"
 
 if [ "$WERCKER_RESULT" = "failed" ]; then
```
Comment (nadade) [inline run.sh]: we're setting the MESSAGE and FALLBACK here again for the failed builds (it's done for both scenarios a few lines above)

**Review by nadade — CHANGES_REQUESTED** [review]
- see my comments
- more:
    - missing a general message for a 500 result (why not just pass the message from slack API if there's error?)
    - missing an option to specify icon_emoji (e.g. :arnold:) - see https://api.slack.com/methods/chat.postMessage

File: run.sh (hunk 1 above)
Comment (abtris) [reply run.sh]: bug, fixed

File: README.md [hunk 3]
```diff
@@ -39,6 +39,34 @@ The MIT License (MIT)
 
 # Changelog
 
+## 1.6.4
```
Comment (nadade) [inline README.md]: 1.6.5?

File: README.md [hunk 4]
```diff
@@ -39,6 +39,34 @@ The MIT License (MIT)
...
+
+- As ID build using `WERCKER_GIT_COMMIT`
```
Comment (nadade) [inline README.md]: using `WERCKER_GIT_COMMIT` as build ID

File: README.md [hunk 5]
```diff
@@ -39,6 +39,34 @@ The MIT License (MIT)
...
+- fix typo
+
+## 1.6.4
+
+- Better hash formating (added # char)
+
+## 1.6.3
+
+- Using `WERCKER_RUN_ID` instead `WERCKER_BUILD_ID` that is only in build workflow
```
Comment (nadade) [inline README.md]: Using `WERCKER_RUN_ID` instead of `WERCKER_BUILD_ID`as that is only available in the build pipeline

File: README.md [hunk 6]
```diff
@@ -39,6 +39,34 @@ The MIT License (MIT)
...
+- Added default Sucess for `WERCKER_RESULT` if notifier isn't run in after steps.
```
Comment (nadade) [inline README.md]: s/Sucess/Success

**Review by nadade — APPROVED** [review]

**Comment by abtris** [comment]
I'm not sure if we can merged or just live in branch.

**Comment by nadade** [comment]
how about creating a production and master branch as with other forks? so one can be in sync with upstream and the other one with our commits...

**Comment by abtris** [comment]
Yes, that can be better approach. I will create production branch from this.

**Pull request closed without merging**